src/elements/content-sidebar/BoxAISidebarContent.tsx

```diff
@@ -23,6 +23,7 @@ import messages from '../common/messages';
 import sidebarMessages from './messages';
 
 import './BoxAISidebar.scss';
+import Tooltip, {TooltipPosition} from "../../components/tooltip/Tooltip";
```

Comment:
I think we can use `Tooltip` component from `blueprint` and I would add storybook interaction test to check if it focuses correctly with `tab` click if it's possible. what do you think?